client/src/components/Login.js

```diff
@@ -24,15 +24,15 @@ const Login = () => {
         const token = data.token;
         localStorage.setItem("token", token);
         localStorage.setItem("user", JSON.stringify(data.user));
-        if (data.user.role === "applicant") {
-          setSuccessRoute("/applicant_create_profile");
-        } else {
-          setSuccessRoute("/public_applicant_profiles");
+        if (data.user && data.user.role === "applicant") {
+          setRedirectRoute("/applicant_create_profile");
+        } else if (data.user && data.user.role === "recruiter") {
+          setRedirectRoute("/public_applicant_profiles");
         }
       })
       .catch(() => {
         setError(true);
-        setSuccessRoute(false);
+        setRedirectRoute(false);
```

Comment:
This should set to empty string instead of `false` 😄 
`setRedirectRoute("");`

Reply:
Changed. Thanks for your assistance, Mohammed.